File: nipuna-ionic-plugin/TestPlugin.js
var exec = require('cordova/exec');

var TestPlugin = {
	startVideoPlayer: function (success, fail, data){
            return exec(success, fail,
                    "TestPlugin",
                    "printStuff",
                     data);
    },
    showAlert: function (success, fail, data){
            return exec(success, fail,
                    "TestPlugin",
                    "showAlert",
                     data);
    }
};

module.exports = TestPlugin;

File: nipuna-ionic-plugin/TestPlugin.m
#import "TestPlugin.h"
#import "TestStoryViewController.h"


@implementation TestPlugin

-(void) printStuff:(CDVInvokedUrlCommand *) command
{
    NSLog(@"This is my ass!");
}

-(void) showAlert:(CDVInvokedUrlCommand *) command{

	NSLog(@"This is sparta!");
	UIStoryboard *mainStoryboard = [UIStoryboard storyboardWithName:@"TestStoryboard"
                                                             bundle: nil];
    
    TestStoryViewController *controller = (TestStoryViewController*)[mainStoryboard
                                                               instantiateViewControllerWithIdentifier:@"TestStoryboard"];
}
@end
